packages/renderless/src/pager/index.ts

```
@@ -68,6 +68,17 @@
     return null
   }
 
+export const computedInternalPageSize =
+  ({ props }: Pick<IPagerRenderlessParams, 'props'>) =>
+  (): number => {
+    const { pageSize, pageSizes } = props
+    let internalPageSize = isNaN(pageSize) ? 10 : Number(pageSize)
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)